lib/Runtime/Library/EngineInterfaceObject.h

```diff
@@ -6,6 +6,28 @@
 
 #if defined(ENABLE_INTL_OBJECT) || defined(ENABLE_JS_BUILTINS)
 
+
+#pragma warning(push)
+#pragma warning(disable:4309) // truncation of constant value
+#pragma warning(disable:4838) // conversion from 'int' to 'const char' requires a narrowing conversion
+
+#if DISABLE_JIT
+#if TARGET_64
+#include "InJavascript/JsBuiltIn.nojit.bc.64b.h"
+#else
+#include "InJavascript/JsBuiltIn.nojit.bc.32b.h"
+#endif // TARGET_64
+#else
+#if TARGET_64
+#include "InJavascript/JsBuiltIn.bc.64b.h"
+#else
+#include "InJavascript/JsBuiltIn.bc.32b.h"
```

Comment:
There used to be 2 byte code headers, 1 for Intl and 1 for JsBuiltins - now combined hence moved the loading to this shared location.